src/main/java/com/uber/kafkaSpraynozzle/KafkaSpraynozzle.java

```diff
@@ -66,40 +67,21 @@ public static void main(String[] args) throws Exception {
         }
 
         if (spraynozzleHA) {
+            buffering = true;
```

Comment:
Do we want implicit, or abort if explicitly misconfigured?

This could produce a nasty surprise if someone has spraynozzle in HA mode off for a few days and then it slams their workers with all of the missed data.